admin/nload/Makefile

```diff
@@ -0,0 +1,40 @@
+# SPDX-Identifier-License: GPL-2.0-only
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nload
+PKG_VERSION:=0.7.4
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://www.roland-riegel.de/nload/
```

Comment:
I'm thinking, don't we want to use secure way to download source files by using [nload's GitHub repository](https://github.com/rolandriegel/nload)?

Reply:
Sure, I update source url to GitHub.